components/library/libmagic/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+# Note: The primary component we deliver initially is libmagic here
+# There is already a "file" utility with true Unix legacy in our OS
+# Still, for the sake of completeness and to match GNU/Linux userland
+# we ALSO deliver a "gfile" built from darwinsys sources, as a bonus.
+COMPONENT_NAME=		file
+COMPONENT_FMRI=		library/magic
+COMPONENT_VERSION=	5.25
+IPS_COMPONENT_VERSION= 5.25.0
```

Comment:
I thought 3-segment versions are required? And note that this is not "5.2.5" - it really is 25th minor in the 5th major series ;)